src/gui-wizard-gtk/wizard.c

```diff
@@ -1162,7 +1162,7 @@ void update_gui_state_from_problem_data(int flags)
     }
     else
     {
-        int count = libreport_xatoi(count_str);
+        int count = g_ascii_strtoll(count_str, NULL, 10);
```

Comment:
`strtoll` converts to a 64-bit integer, which is wider than `int`. (Same with the other variations you used.)

Reply:
So, a different function, or just cast to `int`?

Reply:
It appears GLib doesn't have a function for 32-bit integers, so one option would be to load the value into a `long`, check all the possible errors (overflow, underflow, errors from `g_ascii_strtoll`) and finally cast it to `int`. The other option is to stick with `libreport_xatoi` for now. It seems to be adequate for the task at hand.

Reply:
Like [this](https://github.com/abrt/libreport/pull/636/commits/316297054a57f81578a26af28724d30b3a22374f)? (I left the tests as they were for the time being as I'll likely remove them altogether - there's no point in testing whether `g_ascii_strtoll` works)

Reply:
Well, yes, but it makes sense to wrap that up in a function and then we're back to `libreport_xatoi`…